src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java

```diff
@@ -226,7 +234,8 @@ protected CronJob createCronJob(ScheduleRequest scheduleRequest) {
 		CronJob cronJob = new CronJobBuilder().withNewMetadata().withName(scheduleRequest.getScheduleName())
 				.withLabels(labels).withAnnotations(this.deploymentPropertiesResolver.getJobAnnotations(schedulerProperties)).endMetadata()
 				.withNewSpec().withSchedule(schedule).withConcurrencyPolicy(concurrencyPolicy).withNewJobTemplate()
-				.withNewSpec().withNewTemplate().withNewMetadata().addToAnnotations(annotations).addToLabels(labels)
+				.withNewSpec().withTtlSecondsAfterFinished(ttlSecondsAfterFinished)
```

Comment:
We will need to come up w/ a good default in the (yet to be added) `this.properties.getCron().getTtlSecondsAfterFinishedString();` or not set the ttl value on the builder when its null as I believe the underlying builder will not like a null value here.